.flake8

```
@@ -2,3 +2,7 @@
 per-file-ignores =
    volunteer_planner/settings/*.py: F403, F405,
    volunteer_planner/settings/__init__.py: F403,
+
+max-line-length = 88
```

For `B950`, line length needs to be set to 80, in order to get a warning for lines longer than 110% of this (ie. longer than 88). For `E501`, `max-line-length` needs to be set to 88.

```suggestion
max-line-length = 80
```

---

*meeeh*